cadquery/cq.py

```diff
@@ -3919,20 +3919,21 @@ def text(
         """
         Create a 3D text
 
-        :param str txt: text to be rendered
-        :param distance: the distance to extrude, normal to the workplane plane
+        :param self:
+        :type self: Workplane
```

Comment:
```suggestion
```
At the moment `self` does not appear in the signature above this (in the HTML docs), and the return type is `~T`. It is less confusing to just leave it off.

Reply:
Thank you for raising this issue regarding `:param self`!  This was my workaround for the issue where the Parameters section is duplicated in the HTML docs.  I will raise this as a separate issue as it affects many other methods (see for example Workplane.box, cboreHole, center, chamfer, circle to name a few).  Perhaps there is another way to solve it.

Reply:
Ah, ok, I thought the duplicates were due to indentation or something.

Please do raise the issue, a general solution would be better than manually documenting every self.